examples/deepseek_r1_example.py

```
@@ -0,0 +1,104 @@
+    result = await review_chain.ainvoke(
+        {"pull_request": retriever.pull_request}, include_run_info=True
+    )
+    return result
+
+def generate_report():
+    """Generate a complete PR report with both summary and code review"""
+    start_time = time.time()
+    
+    # Run the summary and review processes
+    summary_result = asyncio.run(pr_summary())
+    print(f"Summary generated successfully")
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)